plugins/catalog/src/routes.ts

```diff
@@ -13,4 +13,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-export { JobStatusModal } from './JobStatusModal';
+
+import { createExternalRouteRef } from '@backstage/core';
+
+export const createComponentRouteRef = createExternalRouteRef({
+  id: 'create-component',
```

Comment:
nit: this can create MORE than just components, can't it? i.e., you could point it to an entity definition for a system and use it that way? wonder if we want to avoid a specific kind as an ID here. i forget where this ID is used...

Reply:
The id itself is just for information in logs and error messages, but it's still a valid point as the name is used in the plugin externalRoutes as well. It's actually only able to create components atm, and the button on the catalog page says "Create Component", which is why we went with this. Maybe could be better to call it just `create`